tesseract_collision/src/create_convex_hull.cpp

```diff
@@ -23,13 +23,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include <tesseract_core/macros.h>
-TESSERACT_IGNORE_WARNINGS_PUSH
+#include <tesseract_collision/core/macros.h>
```

Comment:
I created the macro to make it easier to add different compiler warnings without having to modify every header or source file. Now you just have to modify the macro and all files get updated. I have been basically wrapping all externally includes inside this macro so we don't get warnings we don't have control of.

My thinking was was to keep the macros separate, mainly because every file most likely will include the macro but may not need the utils. Other than that I don't have a strong opinion either way.

>   We really should figure out something for the documentation. I would have never known this tool was here.

I agree, The tesseract_collision package for the most part after this change should not change a lot going forward.

Reply:
Oh, I meant the whole create_convex_hull executable

Reply:
I was just looking at the line. The code that creates the convex hull is in the `core/common.h` but this is just a wrapper around it to allow for it to be used from the command line.